icu4c/source/common/.vscode/settings.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
Definitely want to exclude this file from your pull-request. :)

(I think we might want to exclude the ".vscode" folder using the .gitignore file in the future. Don't worry about that now though, that can be a totally separate PR later on)